PeerioTechnologies/peerio-desktop
# fix: MembersSection in channel sidebar causing screen of death

#### Relevant info and issue/PR links
https://app.clubhouse.io/peerio/story/17021/mc-desktop-error-on-sign-in

#### Testing instructions
1. log in with a user with the role of MC Admin (or "clerk")
2. go to any room inside a patient space
3. on dev or master, this will cause orange screen of death. on this branch it should not.


----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
